packages/11ty/_includes/components/object-filters/object-card/object-image.webc

```diff
@@ -9,23 +9,38 @@
 const alt = ({ data }) => typeof data.thumbnail === 'object' ? data.thumbnail.alt : '';
 
 /**
- * Get object image `src` from `figures` or `thumbnail` properties
+ * Returns the served asset location for this src
+ **/
+const assetSrc = (srcPath) => {
+  if ( srcPath.startsWith('/iiif') || 
+          srcPath.startsWith('http://') || 
+          srcPath.startsWith('https://') ) return srcPath
+
+  // TODO: Account for pathPrefix here
+  // Everything else is a hosted asset
+  console.log(`${this.config.figures.imageDir}/${srcPath}`)
+  return `${this.config.figures.imageDir}/${srcPath}`
+}
```

Comment:
💭 writing this as a more compact expression may subjectively make it more readable
```suggestion
 * Returns the served asset location for this image
 * accounting for fully qualified asset URLs
 * @TODO account for pathPrefix of hosted assets
 */
const assetSrc = (srcPath) => {
  const regexp = new RegExp(/^https?:\/\/|\/iiif\//)
  const { imageDir } = this.config.figures
  return regexp.test(srcPath)) ? srcPath : `${imageDir}/${srcPath}`
}
```
though as elsewhere we might also turn the srcPath tests  into a separate util module that can be called here in just one line like so (note bene the boolean is reversed from the implementation above)
```
  return isAssetHosted(srcPath) ? `${imageDir}/${srcPath}` : srcPath;
```